active/0000-daemonize.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: 2015-05-26
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+ember-cli will run as a daemon in the background, rather than being started for each `ember` command.
+
+# Motivation
+
+Currently running an `ember` command can take quite some time, especially when a lot of addons are installed.
+This is mainly due to the "boot up" period, when Node.js / io.js loads ember-cli itself and *all* the addons.
+Node.js / io.js unnecessarily compiles / loads a huge amount of files that very likely aren't relevant to the task.
```

Comment:
At least this is what I experienced. Running an `ember` command gets slower the more addons I have installed.

However, this could just be me, as I'm running ember-cli in a Ubuntu VM, which may degrade performance to a point, where this effect is noticeable.